src/migrations/cluster-store/snap.ts

```diff
@@ -0,0 +1,30 @@
+// Fix embedded kubeconfig paths under snap config
+
+import { migration } from "../migration-wrapper";
+import { ClusterModel, ClusterStore } from "../../common/cluster-store";
+import { getAppVersion } from "../../common/utils/app-version";
+
+export default migration({
+  version: getAppVersion(), // Run always after upgrade
+  run(store, printLog) {
+    if (!process.env["SNAP"]) return;
+
+    printLog("Migrating embedded kubeconfig paths")
+    const storedClusters: ClusterModel[] = store.get("clusters") || [];
+    if (!storedClusters.length) return;
+
+    printLog("Number of clusters to migrate: ", storedClusters.length)
+    const migratedClusters = storedClusters
+      .map(cluster => {
+        /**
+         * replace snap version with 'current' in kubeconfig path
+         */
+        const kubeconfigPath = cluster.kubeConfigPath.replace(/\/snap\/kontena-lens\/[0-9]*\//, "/snap/kontena-lens/current/")
```

Comment:
Should we only do this if file does not exist?

Reply:
Added check for that